CHANGELOG.md

```diff
@@ -46,6 +46,14 @@ For older changes see the [archived Singularity change log](https://github.com/a
 - Add configuration files binding for `--nv` option, these files are added in
   the `nvliblist.conf` that are recognized in the NVIDIA Container Toolkit,
   including files for EGL ICD.
+- `apptainer.conf` now accepts setting the `allow uts ns` option, and can
+  invalidate the use of the `--uts` and `--hostname` flags.
```

Comment:
This is a duplicate of the message below

Reply:
thanks, I've revisited the changelog entries.